src/components/MovieItem.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <router-link :to="{ name: 'Movie', params: { movieId: movie.id } }">
```

Comment:
Great use of named routes and passing parameters here!

Have you tried to use it in action somewhere, because I'm not able to find it used anywhere 🤔